VRDR/IJEMortality.cs

```diff
@@ -286,6 +286,27 @@ private string DateTime_Get(string ijeFieldName, string dateTimeType, string fhi
             }
         }
 
+        /// <summary>Get the date part value from DateOfBirthDatePartAbsent if it's populated</summary>
+        private string BirthDate_Part_Absent_Get(string ijeFieldName, string dateType, string dateAbsentType)
```

Comment:
I don't understand this (due to my ignorance), so we need a review by someone who does ;-)